src/open_inwoner/components/templates/components/Form/Error.html

```diff
@@ -1,2 +1,10 @@
-{% load notification_tags %}
-{% notifications messages=messages compact=True %}
+{% load i18n button_tags icon_tags button_tags icon_tags notification_tags %}
+
+<div class="notifications notifications__error">
+   <div class="notification notification__error notification--compact">
+    {% for message in messages %}
+        {% icon icon="warning_amber" icon_position="before" outlined=True %}
+        {% notification icon=message.icon icon_position="before" type=message.level_tag message=message.message closable=False compact=compact %}
```

Comment:
Why do we add `closable=False` here? Don't we want the user to be able to close the notification?

Reply:
@vaszig I need a bit of help with differentiation (maybe add a new 'warning' template instead of an error template?); 
because most of the time we actually don't need the close-button at all. Like: if a user forgets to fill a required field; then the user just needs to do that and re-submit; clicking away errors would have no function in that case. 
I am not sure which forms/errors actually need a close function and how much validation we have. 
Perhaps we only need a close-button in the Inbox Messages field?

Reply:
After consultation with @Bartvaderkin the decision is to make the Error template more static, and to not have a close-button.